src/ravenpy/ravenpy.py

```diff
@@ -296,8 +296,15 @@ def run(
     if not outputdir.is_absolute():
         outputdir = (configdir / outputdir).absolute()
 
-    if overwrite and outputdir.exists():
-        shutil.rmtree(str(outputdir))
+    results = [f for f in outputdir.rglob(f"{modelname}_*.*")]
```

Comment:
Files always start with the model name?

Reply:
If `modelname` is provided, then yes. I've added an option for when it isn't (e.g. `Hydrographs.nc`).